docs/docs/headless-cms.md

```diff
@@ -15,6 +15,20 @@ The guides in this section will walk through the process of setting up content s
 
 [[guidelist]]
 
+Other CMS systems you can connect to include:
+
+* [Shopify](https://www.gatsbyjs.org/packages/gatsby-source-shopify/?=gatsby-source-)
```

Comment:
I don't think it actually matters, but should probably remove the query string at the end of all of these too.

Reply:
I actually included it on purpose so the user sees only Gatsby sources when they click through. Happy either way though.